test cases/common/229 persubproject options/subprojects/sub2/foo.c

```diff
@@ -1,5 +1,9 @@
 int foo(void);
 
+#ifdef __GNUC__
```

Comment:
Doesn't this mean the test only actually works on glibc?

Reply:
The CI fails with MSVC that doesn't support `#warning`. Since this change is compiler agnostic, it doesn't matter, IMO.

Reply:
I guess thats fair